scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java

```diff
@@ -952,6 +979,22 @@ public String getTaskServerLogs(String jobId, String taskName) throws UnknownJob
         throw new UnknownTaskException("Unknown task " + taskName + " in job " + jobId);
     }
 
+
+    @Override
+    @ImmediateService
+    public String getTaskServerLogsByTag(String jobId, String taskTag) throws UnknownJobException, NotConnectedException, PermissionException {
+        JobId id = JobIdImpl.makeJobId(jobId);
+        frontendState.checkJobOwner("getTaskServerLogs", id,
```

Comment:
yes, and additionally the new methods getTaskServerLogsByTag, getTaskResultsByTag etc must be added in the security.java.policy-server file , otherwise things will not work properly